app/assets/stylesheets/search.scss

```diff
@@ -0,0 +1,45 @@
+.searchbar {
+  background: #dee0e2;
+  padding: 0.5em;
+  margin-bottom: 20px;
+
+  .form-group {
+    margin-bottom: 15px;
+  }
+  .form-group-error {
+    margin-right: 0;
+  }
+}
+
+.searchfield {
+  position: relative;
+}
+
+input[type="text"].searchfield--input {
+  font-family: "proxima_nova", Arial, sans-serif;
+  font-weight: 400;
+  text-transform: none;
+  line-height: 1.25;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  width: 100%;
+  margin: 0 0 15px;
+  padding: 0 0 0 10px;
+  display: block;
+  height: 46px;
+  border-color: #0b0c0c;
+}
+
+input[type="search"] {
+  -webkit-appearance: textfield;
+  -moz-box-sizing: content-box;
+  -webkit-box-sizing: content-box;
+  box-sizing: content-box;
+}
+
+@media (min-width: 641px) {
+  main.search .search-header input {
+    line-height: 1.31579;
+  }
+}
```

Comment:
Why is this file duplicated? 
app/assets/css/search.scss   and   app/assets/stylesheets/search.scss